client/state/timezones/actions.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Internal dependencies
+ */
+import {
+	TIMEZONES_RECEIVE,
+	TIMEZONES_REQUEST,
+	TIMEZONES_REQUEST_FAILURE,
+	TIMEZONES_REQUEST_SUCCESS
+} from 'state/action-types';
+
+export const timezonesRequestAction = () => ( {
+	type: TIMEZONES_REQUEST
+} );
+
+export const timezonesRequestSuccessAction = () => ( {
+	type: TIMEZONES_REQUEST_SUCCESS
+} );
+
+export const timezonesRequestFailureAction = error => ( {
+	type: TIMEZONES_REQUEST_FAILURE,
+	error
+} );
+
+/*
+ * The data given in as the function parameter has already been checked
+ * from the `timezones` data-layer.
```

Comment:
aha! here lies some beauty - this data doesn't have to come from the data-layer (and actually then needs no reference here)

validation in the reducers is different than validation from the API

 - the data-layer makes sure that the information returned from the API fits the right _shape_ and _structure_ of the data that Calypso expects
 - the Redux system makes sure that the information it receives _makes sense_ and is valid in the terms of things like acceptable values

we could just as easily send this request from the dev tools or over a web socket or have them included in a JSON blob at the top of the original HTML